src/components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import visaImg from "../../assets/images/visa.png"
+import masterCardImg from "../../assets/images/mc_symbol_opt_63_2x.png"
+import './CreditCard.css'
+
+export default function CreditCard({type, number, expirationMonth, expirationYear, bank, owner, bgColor, color}) {
+  const cardLogo = type === "Visa" ? visaImg : masterCardImg;
+  if(expirationMonth <10){
+    expirationMonth = "0" + expirationMonth
+}
+const divStyle = {
+  backgroundColor: bgColor,
+  color: color,
+  }
+  return (
+    <div className="creditCard" style={divStyle}>
+      <div className="logoBanco">
+        <img src={cardLogo} alt="card logo" style={{width: '80px', height: '40px'}} />
```

Comment:
```suggestion
        <img src={cardLogo} alt="card logo" />
```

Te recomiendo que no lo hagas, ademas en el css que has puesto ya se lo estabas dando asi que vamos no tiene sentido

Reply:
tienes razon lo puse doble